ui/articles-aem-renderer/src/main/java/org/godtools/articles/aem/db/ArticleRepository.java

```diff
@@ -0,0 +1,59 @@
+package org.godtools.articles.aem.db;
+
+import android.app.Application;
+import android.arch.lifecycle.LiveData;
+import android.os.AsyncTask;
+
+import org.godtools.articles.aem.model.Article;
+
+import java.util.List;
+
+/**
+ * This class is use to go connection to the Article Data Access Object.
+ *
+ * @author Gyasi Story
+ */
+public class ArticleRepository {
+
+    private ArticleDao mArticleDao;
+    private LiveData<List<Article>> mAllArticles;
+
+    public ArticleRepository(Application application) {
+        ArticleRoomDatabase db = ArticleRoomDatabase.getINSTANCE(application);
+        mArticleDao = db.mArticleDao();
+        mAllArticles = mArticleDao.getAllArticles();
+    }
+
+    /**
+     * To call a list of all article stored in the database.
+     *
+     * @return = Live Data Collection of Articles
+     */
+    public LiveData<List<Article>> getmAllArticles() {
```

Comment:
`getAllArticles()`

guessing this was autogenerated, there is a way to configure AS to ignore the `m` prefix when autogenerating getters/setters

Reply:
it's under Android Studio -> Preferences -> Editor -> Code Style -> Java -> Code Generation.

add "m" to the Field name prefix.
![screen shot 2018-08-24 at 11 47 48 am](https://user-images.githubusercontent.com/47818/44594275-b300b800-a793-11e8-98b6-6d9069f11a42.PNG)